### PR TITLE
app-laptop/tuxedo-keyboard add 3.1.3

### PR DESCRIPTION
**Pull request by LukashonakV**
Signed-off-by: Viktar Lukashonak <myxabeer@gmail.com>

### CODE REVIEW

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
```
Comment (scardracs) [inline app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:1]: 1999-2023

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+inherit linux-mod toolchain-funcs
+
+DESCRIPTION="Kernel Module for Tuxedo Keyboard"
+HOMEPAGE="https://github.com/tuxedocomputers/tuxedo-keyboard"
+SRC_URI="https://github.com/tuxedocomputers/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (scardracs) [inline app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:16]: Drop it

Comment (LukashonakV) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:16]: Done

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild (hunk 1 above)
Comment (LukashonakV) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:1]: Done

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild (hunk 2 above)
Comment (LukashonakV) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild]: I dropped, but QA CI complains on LICENSING. So need to bring it back.

**Comment by scardracs** [comment]
Please squash commits. It looks good to me but actually I'm not a dev so I can't do anything for you. You'll have to wait for a dev

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild (hunk 2 above)
Comment (LukashonakV) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:16]: "SLOT="0" is a mandatory variable according https://devmanual.gentoo.org/ebuild-writing/variables/index.html

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (scardracs) [inline app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:4]: Uh, I didn’t notice it. EAPI=8

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild (hunk 2 above)
Comment (scardracs) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:16]: You have to drop only the empty IUSE

**Review by scardracs — CHANGES_REQUESTED** [review]

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild (hunk 3 above)
Comment (mschiff) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:4]: EAPI=7 is ok for now as we do not use EAPI 8 features and 7 is still supported.

**Review by mschiff — APPROVED** [review]
looks OK to me

**Comment by mschiff** [comment]
Please signoff your changes so I can merge your PR, thanks!

**Comment by LukashonakV** [comment]
Done. Thank you

File: app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild (hunk 3 above)
Comment (scardracs) [reply app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild:4]: Which one does not have EAPI=8? Both [linux-mod](https://github.com/gentoo/gentoo/blob/master/eclass/linux-mod.eclass#L10) and [toolchain-funcs](https://github.com/gentoo/gentoo/blob/master/eclass/toolchain-funcs.eclass#L7) have it enabled?

**Comment by scardracs** [comment]
> Done. Thank you

`IUSE=""` is still here, could you drop it? That what I meant at first but GH tends to take a slice of code and that leads to you to the error did before

**Comment by LukashonakV** [comment]
Hi @scardracs , sure. Done. Thank you

**Pull request closed without merging**